### PR TITLE
Update and add fields for new registration API

### PR DESCRIPTION
**Pull request by mkangia**
[Changes](https://docs.google.com/spreadsheets/d/1LK6xQHldOT8tf3ctcypAyUlIuTN-C25zhyIC1jHJeS0/edit#gid=917275059)

### CODE REVIEW

File: custom/enikshay/integrations/nikshay/repeater_generator.py [hunk 1]
```diff
@@ -93,8 +94,10 @@ def _base_properties(self, repeat_record):
             "RegBy": username,
             "password": password,
             "Source": ENIKSHAY_ID,
+            "source": ENIKSHAY_ID,
             "IP_From": server_ip,
             "IP_FROM": server_ip,
+            "ip_address": server_ip,
```
Comment (proteusvacuum) [inline custom/enikshay/integrations/nikshay/repeater_generator.py]: lol

**Review by proteusvacuum — COMMENTED** [review]
looking like a good start! no major comments.

**Review by proteusvacuum — APPROVED** [review]

File: custom/enikshay/integrations/nikshay/field_mappings.py [hunk 2]
```diff
@@ -156,3 +157,25 @@
     'public': 'P',
     'private': 'V',
 }
+
+marital_status = {
+    'unmarried': 'S',
+    'married': 'M',
+    'widowed': 'W',
+    'separated': 'Se'
```
Comment (mkangia) [inline custom/enikshay/integrations/nikshay/field_mappings.py]: W and Se should just become 'other'.

**Comment by proteusvacuum** [comment]
let me know when you want me to review this again

**Comment by mkangia** [comment]
sure @proteusvacuum 

this is not intended to be available as a repeater anymore i.e this does not need to check on updates on cases and notify. We would just get a list of episode ids and then just notify them using the payload generator. Would remove the repeater and other related code. Any inputs/remarks?

**Comment by proteusvacuum** [comment]
:cry:

**Pull request closed without merging**